sonic_api.yaml

```diff
@@ -120,6 +121,7 @@ paths:
 #----------------------------------------------
   '/state/heartbeat':
     get:
+      operationId: StateHeartbeatGet
```

Comment:
What is operationId used for?

Reply:
operationId is used by autorest to generate the function names for these apis